File: insight/src/test/testData/javascript/MultiInsight.js
function probabilityAndDuration() {
    if (Math.random() > 0.5) { //sleep 100ms
        if (Math.random() > 0.5) { //sleep 100ms
            console.log(false); //sleep 200ms
        }
    } else {
        console.log(true); //sleep 200ms
    }
}

File: insight/src/test/testData/java/MultiInsight.java
public class MultiInsight {
    public void probabilityAndDuration() {
        if (java.lang.Math.random() > 0.5) { //sleep 100ms
            if (java.lang.Math.random() > 0.5) { //sleep 100ms
                Thread.sleep(200);
            }
        } else {
            Thread.sleep(200);
        }
    }
}